modules/mono/glue/GodotSharp/GodotSharp/Compat.cs

```diff
@@ -30,13 +30,15 @@ public double BlendNode(StringName name, AnimationNode node, double time, bool s
 
 partial class AnimationPlayer
 {
+#if !DISABLE_DEPRECATED
```

Comment:
We'd want to disable the entire `Compat.cs` file, since all of these APIs only exist for compatibility.